module2/test_calculator.py

```diff
@@ -0,0 +1,42 @@
+import unittest
+class Calculator:
+    def add(self, x, y):
+        return x + y
+
+    def subtract(self, x, y):
+        return x - y
+
+    def multiply(self, x, y):
+        return x * y
+
+    def divide(self, x, y):
+        if y == 0:
+            return "Error! Division by zero."
+        return x / y
+
```

Comment:
The unitest code should not be included module code. It should be in separate file.     Your module:   **calculator.py**   unit test code file : **test_calculator.py**

Reply:
ok, got the point , i will modify and update the code